data/sql/updates/pending_db_world/Scarlet_Enclave_Phase_4.sql

```diff
@@ -0,0 +1,559 @@
+
+-- Wander Distance and Movement Type
+
+UPDATE `creature` SET `wander_distance` = 0, `MovementType` = 2 WHERE `guid` IN (129674, 129675, 129672, 129677, 129673, 129681, 129667, 129678, 129671, 129679) AND `id1` = 28939;
+
+
+-- Paths
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (129677, 129673, 129674, 129672, 129675, 129678, 129679, 129671, 129681, 129667));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(129677, 12964600, 0, 0, 1, 0, 0, ''),
+(129673, 12965500, 0, 0, 1, 0, 0, ''),
+(129674, 12963900, 0, 0, 1, 0, 0, ''),
+(129672, 12964800, 0, 0, 1, 0, 0, ''),
+(129675, 12965100, 0, 0, 1, 0, 0, ''),
+(129678, 12965600, 0, 0, 1, 0, 0, ''),
+(129679, 12963800, 0, 0, 1, 0, 0, ''),
+(129671, 12964000, 0, 0, 1, 0, 0, ''),
+(129681, 12965300, 0, 0, 1, 0, 0, ''),
+(129667, 12964300, 0, 0, 1, 0, 0, '');
+
+
+-- SCARLET CRUSADERS
+
+-- Positions
+
+UPDATE `creature` SET `position_x` = 1479.81, `position_y` = -5878.11, `position_z` = 132.03, `orientation` = 3.4523 WHERE `guid` = 129723 AND `id1` = 28940;
+
+-- Wander Distance and Movement Type
+
+UPDATE `creature` SET `wander_distance` = 5, `MovementType` = 1 WHERE `guid` = 129723 AND `id1` = 28940;
+UPDATE `creature` SET `wander_distance` = 0, `MovementType` = 2 WHERE `guid` IN (129702, 129722) AND `id1` = 28940;
+
+-- Paths
+
+UPDATE `creature_addon` SET `path_id` = 12970200, `auras` = 52103 WHERE (`guid` IN (129702));
+UPDATE `creature_addon` SET `path_id` = 12972200, `auras` = 52103 WHERE (`guid` IN (129722));
+
+
+-- SCARLET MARKSMEN
+
+-- Wander Distance and Movement Type
+
+UPDATE `creature` SET `wander_distance` = 0, `MovementType` = 2 WHERE `guid` IN (130009, 130014) AND `id1` = 28610;
+
+-- Paths
+
+UPDATE `creature_addon` SET `path_id` = 13000900, `auras` = 52103 WHERE (`guid` IN (130009));
+UPDATE `creature_addon` SET `path_id` = 13001400, `auras` = 52103 WHERE (`guid` IN (130014));
+
+
+-- Random Spawn for Scarlet Commanders/Preachers with Waypoints
+
+DELETE FROM `pool_template` WHERE (`entry` IN (22346, 22347, 22348, 22349, 22350, 22351, 22352, 22353, 22354, 22355));
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES 
+('22346', '1', 'Scarlet Commander/Preacher Spawn (1 out 10)'),
+('22347', '1', 'Scarlet Commander/Preacher Spawn (2 out 10)'),
+('22348', '1', 'Scarlet Commander/Preacher Spawn (3 out 10)'),
+('22349', '1', 'Scarlet Commander/Preacher Spawn (4 out 10)'),
+('22350', '1', 'Scarlet Commander/Preacher Spawn (5 out 10)'),
+('22351', '1', 'Scarlet Commander/Preacher Spawn (6 out 10)'),
+('22352', '1', 'Scarlet Commander/Preacher Spawn (7 out 10)'),
+('22353', '1', 'Scarlet Commander/Preacher Spawn (8 out 10)'),
+('22354', '1', 'Scarlet Commander/Preacher Spawn (9 out 10)'),
+('22355', '1', 'Scarlet Commander/Preacher Spawn (10 out 10)');
```

Comment:
the integers could be converted from string to int (same below)

Reply:
Lol, you are right. It was a direct copy/paste from Mysql.

Reply:
Solved.